dom.bs

```diff
@@ -2440,8 +2438,12 @@ before a <var>child</var>, with an optional <i>suppress observers flag</i>, run
  <li><p>Let <var>previousSibling</var> be <var>child</var>'s <a>previous sibling</a> or
  <var>parent</var>'s <a>last child</a> if <var>child</var> is null.
 
+ <li><p>Let <var>deferredStepsQueue</var> be the empty <a>queue</a>.
+
+ <li><p>Let <var>textChanged</var> be false.
+
  <li>
-  <p>For each <var>node</var> in <var>nodes</var>, in <a>tree order</a>:
+  <p>For each <var>node</var> in <var>nodes</var>:
```

Comment:
Optional nit: `<a for=list>For each</a>`

Reply:
Changed that.